stylelint-polaris/plugins/coverage/index.js

```diff
@@ -30,7 +30,7 @@ module.exports = stylelint.createPlugin(
         categoryConfigRules,
       )) {
         rules.push({
-          coverageRuleName: `${ruleName}/${categoryName}`,
+          coverageRuleName: `polaris/${categoryName}/${categoryRuleName}`,
```

Comment:
Do we want only `polaris/colors/...` etc, or the full coverage ruleName?

```suggestion
          coverageRuleName: `${ruleName}/${categoryName}/${categoryRuleName}`,
```

Reply:
Great callout! I received early feedback that including `coverage` in the generated name was redundant and only relevant in the context of our coverage site. It wasn't really worth addressing at the time, but seems appropriate now that the appended rule name's length is unknown.